myhpi/core/templates/core/minutes.html

```diff
@@ -8,69 +8,69 @@
+            <aside class="side-panel border-accent">
+                <h1 class="toc-title">{% translate "Participants" %}</h1>
                 <ul>
-                {% for guest in page.guests %}
-                    <li>{{ guest }}</li>
-                {% endfor %}
+                    {% for participant in page.participants.all %}
+                        <li>{{ participant.get_full_name }}</li>
+                    {% endfor %}
                 </ul>
-            {% else %}
-                <i>{% translate "No guests" %}</i>
-            {% endif %}
-            <h4>{% translate "Labels" %}</h4>
+            </aside>
+            <aside class="side-panel border-accent">
+                <h1 class="toc-title">{% translate "Guests" %}</h1>
+                {% if page.guests %}
+                    <ul>
+                    {% for guest in page.guests %}
+                        <li>{{ guest }}</li>
+                    {% endfor %}
+                    </ul>
+                {% else %}
+                    <i>{% translate "No guests" %}</i>
+                {% endif %}
+            </aside>
+            <aside class="side-panel border-accent">
+                <h1 class="toc-title">{% translate "Labels" %}</h1>
                 {%  if page.labels.all %}
-                    <p>
-                    {% include "core/label.html" with minutes=page %}
-                    </p>
-                    {% else %}
+                    <p>{% include "core/label.html" with minutes=page %}</p>
+                {% else %}
                     <i>{% translate "No labels"%}</i>
                 {% endif %}
-            <h4 class="d-print-none">{% translate "Table of contents" %}</h4>
-            {{ parsed_md.1 }}
-            {% if page.attachments.all %}
-                <h4 class="d-print-none">{% translate "Attachments" %}</h4>
-                <ul>
-                {% for attachment in page.attachments.all %}
-                    <li>
-                        <a href="{{ attachment.url }}">{{ attachment.title }}</a>
-                    </li>
-                {% endfor %}
-                </ul>
-            {% endif %}
+            </aside>
+            {% include "core/sidebar.html" %}
         </div>
         <div id="minutes-footer" class="d-none d-print-block"></div>
-            <div id="minutes-navigation" class="row d-print-none">
-                <div class="col-6">
-                    {%  prev_minutes page as prev %}
-                    {% if prev %}
-                        <a href="{{ prev.url }}">< {% translate "Previous minutes" %}</a>
-                    {%  endif %}
-                </div>
-                <div class="col-6">
-                    {%  next_minutes page as next %}
-                    {% if next %}
-                        <a href="{{ next.url }}"> {% translate "Next minutes" %} ></a>
-                    {%  endif %}
-                </div>
+        <div id="minutes-navigation" class="row d-print-none">
```

Comment:
Not part of this PR, but I'm not really happy with the styling here (Missing space, the arrows look very programmer-art, maybe introduce actual buttons?)
![image](https://github.com/fsr-de/myHPI/assets/6863832/67ea06a7-c6fa-4b5c-b653-07f78f851ef4)

Reply:
Me too. But first make an issue out of it?